ansible/modules/hashivault/hashivault_oidc_auth_role.py

```diff
@@ -85,6 +89,7 @@
     allowed_redirect_uris:
         description:
             - The list of allowed values for redirect_uri during OIDC logins.
+            - When using nested namespaces, use url encoding '%2F' instead of '/'
```

Comment:
If hvac ever supports oidc, this is going to cause an incompatibility that could be avoid with:

```
import urllib.parse
namespace = urllib.parse.quote(namespace, safe='')
```

Reply:
I think this may be a bug with HashiCorp Vault. We actually do not want URL encoding for the namespace. We will get an error along the lines of 
```
* no handler for route 'namespace1%2Fnamespace2/auth/oidc/role/rolename`. 
```
We only need it encoded for the appended namespace part in `allowed_redirect_uris`.
```
```